.github/workflows/ci.yml

```diff
@@ -0,0 +1,79 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+    - "main"
```

Comment:
indent

Reply:
not sure I understand this one, isn't this the same in all our ci.ymls?

Reply:
it was copied like this in our ci.yml, but later on I received PR comments requesting adding the spaces for indention to make it looks like:

> on:
>   pull_request:
>     branches:
>       - "main"
>

Reply:
I just checked again and it seems that the YAML spec doesn't enforce indention or consistency in terms of the usage of spaces, so I guess there is no impact whether we indent or not.
but for the purpose of keeping our code style consistent, I feel it won't hurt to add indention similar to our swagger yml